schema/me/consignments/submissions.js

```diff
@@ -0,0 +1,31 @@
+import { SubmissionType } from "./submission"
+import { GraphQLBoolean } from "graphql/type/scalars"
+
+import { pageable } from "relay-cursor-paging"
+import { connectionDefinitions, connectionFromArraySlice } from "graphql-relay"
+
+export default {
+  type: connectionDefinitions({ nodeType: SubmissionType }).connectionType,
+  args: {
+    ...pageable({}),
+
+    completed: {
+      type: GraphQLBoolean,
+      default: true,
+    },
+  },
+  description: "A list of the current user’s consignment submissions",
+  resolve: (root, options, _request, { rootValue: { submissionsLoader } }) => {
+    // TODO: This should get replaced in stitching,
+    //       it _does not_ handle pagination at API level, so you'll need a
+    //       list size that's a little higher than average while there
+    //       are still small amounts of data.
```

Comment:
I don’t understand what this comment is trying to convey. Is it important for drive-by readers to understand it or a note to self?

Reply:
drive-by - basically, today you can use this but it will only paginate once and think it's done. In the future the stitched version will paginate correctly.